### ISSUE TITLE
Get-TppCertificateDetail Missing Offset

### ISSUE THREAD

**Issue by BeardedPrincess** [gdbarron/VenafiTppPS#26]
The current implementation of Get-TppCertificateDetail allows for the developer to specify a 'limit' parameter, but without the 'offset' parameter. 

The idea is that Limit and Offset should be used together. As an example, if I had a query that returns 103 certificates, with a limit of 10, the first query would be offset 0, limit 10 to get the first 10 records. Then I'd check the total number of records, and decided that I need to fetch the next 10 using Limit 10, Offset 10. You can repeat until you have retrieved all the records.

My suggestion would be to eliminate exposing the offset to the user (it's confusing, and requires them to implement logic). Better would be to allow specifying the limit (as you have done), but detect the total number of records and make as many calls as necessary to return all of the results.

There is sample code implementing similar logic here: https://support.venafi.com/hc/en-us/articles/360001641051

**Comment by gdbarron** [comment]
Thanks, @BeardedPrincess.  Perhaps it's an "undocumented feature", but I noticed that setting a limit of 0 brought back all records in one call.  I've set that to the default for now, but sounds like I'll need to make some changes either way.  My thought is to keep limit and utilize offset as you state and add an All param which would provide a limit of 0 behind the scenes.  If the use of 0 isn't suggested/supported, I'll keep fetching until done, but like the idea of one call.
